.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ workflows:
       - unit_tests:
           matrix:
             parameters:
-              python_version: [ "3.7" ]
+              python_version: [ "3.7", "3.9" ]
```

Comment:
is this added here because we rely on `3.8` for codecov?

Reply:
Yup! We also don't test w/ core dependencies as true